### PR TITLE
Refund script

### PR DESCRIPTION
**Pull request by asood123**

### CODE REVIEW

File: scripts/check_for_refunds_on_stripe.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+/*
+ * This script tells us which Stripe subscriptions are inactive
```
Comment (xdamman) [inline scripts/check_for_refunds_on_stripe.js]: Not sure this comment is valid.

File: scripts/check_for_refunds_on_stripe.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+ */
+
+const app = require('../server/index');
+import models from '../server/models';
+import { retreiveCharge } from '../server/gateways/stripe';
```
Comment (xdamman) [inline scripts/check_for_refunds_on_stripe.js]: "retreiveCharge" typo

File: server/gateways/stripe.js [hunk 3]
```diff
@@ -92,6 +92,13 @@ export const createCharge = (stripeAccount, charge) => {
 };
 
 /**
+ * Fetch charge
+ */
+export const retreiveCharge = (stripeAccount, chargeId) => {
+  return client(stripeAccount).charges.retrieve(chargeId);
```
Comment (xdamman) [inline server/gateways/stripe.js]: one "retrieve" is well written here :-)

**Comment by asood123** [comment]
Good catch, fixed.

**Pull request merged**